Source/WebInspectorUI/UserInterface/Views/CookiePopover.js

```diff
@@ -211,6 +219,9 @@ WI.CookiePopover = class CookiePopover extends WI.Popover
 
         this._pathInputElement = createInputRow("path", WI.unlocalizedString("Path"), "text", data.path).inputElement;
 
+        if (options.includePartitionKey)
+            this._partitionKeyInputElement = createInputRow("partitionKey", WI.unlocalizedString("Partition Key"), "text", data.partitionKey).inputElement;
```

Comment:
This should be a localized string because unlike the other columns, it is not a literal that you'd find in the cookie itself.  In CHIPS it would be `Partitioned`.